src/replicator.js

```diff
@@ -1,7 +1,10 @@
+const path = require('path')
 const EventEmitter = require('events')
 const OrbitDB = require('orbit-db')
 const Pubsub = require('orbit-db-pubsub')
 const AccessControllers = require('orbit-db-access-controllers')
+const Storage = require('orbit-db-storage-adapter')
+const Cache = require('orbit-db-cache')
```

Comment:
Maybe name these `OdbStorage` and `OdbCache` for clarity?

Reply:
good idea